lexibot.py

```diff
@@ -1,30 +1,24 @@
-# -*- python -*-
-# ex: set filetype=python:
-
-from buildbot.plugins import *
+import collections
 
 from github import Github
 
+from buildbot.plugins import *
+
 exclude = ['pylexibank', 'lexibank']
```

Comment:
'mit-chapter'

Reply:
and 'template', 'phylogenetics-data-management-tutorial'

Reply:
Got the latter two. Currently, I only retrieve public repositories - which I guess makes sense?
I'm not sure how we should go about including private repos. Giving buildbot access will require
- generating ssh keys on buildbot
- a buildbot user on github with these keys

So that's a whole lot more maintenance.

Reply:
Hmm, we probably want those monitored too.. pygithub can work with an access token, does that help? I'll make an issue or two in https://github.com/cldf/cldf-buildbot